tests/test_channels/test_channels.py

```diff
@@ -1,8 +1,6 @@
 from tests.base import BaseTestCase
 
-
```

Comment:
Why do you remove this line? Please see PEP * recommendations on blank lines before classes: https://www.python.org/dev/peps/pep-0008/#blank-lines